static/_redirects

```diff
@@ -39,10 +39,10 @@
 
 /docs/try-snowplow/recipes/* /docs/recipes/:splat 301
 
-/docs/getting-started-with-snowplow-bdp/what-is-snowplow-bdp/ /docs/getting-started-with-snowplow-bdp/enterprise/ 301
+/docs/getting-started-with-snowplow-bdp/what-is-snowplow-bdp/ /docs/getting-started-on-snowplow-bdp-enterprise/ 301
 /docs/getting-started-with-snowplow-bdp/what-is-snowplow-bdp/feature-comparison/ /docs/feature-comparison/ 302
-/docs/getting-started-with-snowplow-bdp/aws-setup-guide/* /docs/getting-started-with-snowplow-bdp/enterprise/setup-guide-aws/:splat 301
-/docs/getting-started-with-snowplow-bdp/setup-guide-gcp/* /docs/getting-started-with-snowplow-bdp/enterprise/setup-guide-gcp/:splat 301
+/docs/getting-started-with-snowplow-bdp/aws-setup-guide/* /docs/getting-started-on-snowplow-bdp-enterprise/setup-guide-aws/:splat 301
+/docs/getting-started-with-snowplow-bdp/setup-guide-gcp/* /docs/getting-started-on-snowplow-bdp-enterprise/setup-guide-gcp/:splat 301
```

Comment:
Do we not need to double redirect?

Reply:
We need to support old links pointing to `/docs/getting-started-with-snowplow-bdp/aws-setup-guide/` and the newer links pointing to `/docs/getting-started-with-snowplow-bdp/enterprise/setup-guide-aws/`. I modified these older redirects so that we don’t double-hop.

We also need separate redirects for AWS and GCP here because they used to be spelled differently and now they are not.

So yes, and yes.